lib/logstash/inputs/twitter.rb

```diff
@@ -223,10 +238,15 @@ def configure(c)
     c.access_token = @oauth_token
     c.access_token_secret = @oauth_token_secret.value
     if @use_proxy
-      c.proxy =  {
-        proxy_address: @proxy_address,
-        proxy_port: @proxy_port,
-      }
+      uri = URI.parse('')
```

Comment:
it seems the twitter client docs [suggest you could just place the host and port into the hash](https://github.com/sferik/twitter/blob/master/examples/Configuration.md#using-a-proxy) as we're doing before, what is the need for creating the URI object here?